Blish HUD/_Utils/WindowUtil.cs

```diff
@@ -180,24 +210,39 @@ public enum OverlayUpdateResponse {
             var activeWindowHandle = GetForegroundWindow();
             GameService.Debug.StopTimeFunc("GetForegroundWindow");
 
-            // If Guild Wars 2 is not the focused application, set Blish HUD to
-            // be above Guild Wars 2, but below the next application's window in z-order
             if (activeWindowHandle != gw2WindowHandle && Form.ActiveForm == null) {
                 if (wasOnTop) {
                     Logger.Debug("GW2 is no longer the active window.");
 
-                    var nextHandle = GetWindow(gw2WindowHandle, GW.HWNDPREV);
-                    if (nextHandle != IntPtr.Zero && nextHandle != winHandle) {
-                        SetWindowPos(winHandle, nextHandle, 0, 0, 0, 0, SWP_NOMOVE | SWP_NOSIZE);
-                    }
+                    LostFocusProc(winHandle, gw2WindowHandle, wasOnTop, clientRect, screenPoint);
                 }
+
                 return (OverlayUpdateResponse.WithoutFocus, screenPoint.X == MINIMIZED_POS, 0);
             }
 
+            UpdateProc(winHandle, gw2WindowHandle, wasOnTop, clientRect, screenPoint);
+
             if (!wasOnTop) {
                 Logger.Debug("GW2 is now the active window - reactivating the overlay.");
+
+                SetTransparentLayered(winHandle);
+                SetLayeredWindowAttributes(winHandle, 0, 255, LWA_ALPHA);
```

Comment:
Transparent window bug:
SetLayeredWindowAttributes(winHandle, 0, 0, LWA_COLORKEY);
SetLayeredWindowAttributes(winHandle, 0, 255, LWA_ALPHA);

Reply:
oh! i see what you did there. i was trying to find a solution for Linux too but without success, they all have the black background...

Reply:
oh! someone made an Overlay class for MonoGame on GitHub... now where is it again...